tests/pytests/test_config.py

```diff
@@ -153,14 +154,14 @@ def test_arg_true(arg_name):
         env.stop()
 
     test_arg_true('NOGC')
-    test_arg_true('SAFEMODE')
+    # test_arg_true('SAFEMODE')
```

Comment:
Why was this deleted?

Reply:
@MeirShpilraien This test will fail when run with `CONCURRENT_WRITE_MODE`. Simply not a good test anymore, once `SAFEMODE` was subordinated to `CONCURRENT_WRITE_MODE`.

Reply:
:+1: